### PR TITLE
feat: [merchantapi] new module for merchantapi reports

### PR DESCRIPTION
**Pull request by mpeddada1**
```
python3.9 generation/new_client/new-client.py generate 
--api_shortname=merchantapi 
--proto-path=google/shopping/merchant/reports 
--name-pretty="Merchant API" 
--product-docs="https://developers.google.com/merchant/api" 
--api-description="Programmatically manage your Merchant Center accounts." 
--distribution-name="com.google.shopping:google-shopping-merchant-reports"
```

### CODE REVIEW

**Comment by mpeddada1** [comment]
Stacktrace:
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.3.0:enforce (enforce) on project grpc-google-shopping-merchant-reports-v1beta: 
Error:  Rule 3: org.codehaus.mojo.extraenforcer.dependencies.BanDuplicateClasses failed with message:
Error:  Duplicate classes found:
Error:  
Error:    Found in:
Error:      com.google.api.grpc:proto-google-common-protos:jar:2.23.0:compile
Error:      com.google.shopping.api.grpc:proto-google-shopping-merchant-reports-v1beta:jar:0.0.1-SNAPSHOT:compile
Error:    Duplicate classes:
Error:      com/google/api/MethodSettings$LongRunningOrBuilder.class
Error:      com/google/api/Logging.class
Error:      com/google/api/Usage$1.class
Error:      com/google/api/AuthProvider$1.class
Error:      com/google/api/ProjectProperties$Builder.class
Error:      com/google/api/DocumentationProto.class
Error:      com/google/api/JwtLocation.class
Error:      com/google/api/RoutingRule$Builder.class
Error:      com/google/api/RubySettingsOrBuilder.class
Error:      com/google/api/MonitoringOrBuilder.class
Error:      com/google/api/ResourceDescriptor$2.class
Error:      com/google/api/MetricDescriptor$MetricDescriptorMetadata$1.class
```

File: gapic-libraries-bom/pom.xml [hunk 1]
```diff
@@ -672,6 +672,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.shopping</groupId>
+        <artifactId>google-shopping-merchant-reports-bom</artifactId>
```
Comment (meltsufin) [inline gapic-libraries-bom/pom.xml:677]: I don't think this should go into the libraries bom, since it's not a cloud library,.
@suztomo

Comment (suztomo) [reply gapic-libraries-bom/pom.xml:677]: Good catch.

@mpeddada1 Would you remove the artifacts under com.google.shopping from the BOM?

Comment (meltsufin) [reply gapic-libraries-bom/pom.xml:677]: Can we modify the script such that group IDs that are not `com.google.cloud` do not get added to the bom automatically?

Comment (mpeddada1) [reply gapic-libraries-bom/pom.xml:677]: Yes, thanks for pointing this out! Will remove this from the BOM

**Comment by mpeddada1** [comment]
Closing in favor of https://github.com/googleapis/google-cloud-java/pull/9896

**Pull request closed without merging**